src/interface/meshblock_data.cpp

```diff
@@ -54,6 +54,18 @@ void MeshBlockData<T>::Initialize(
     AddField(q.first.base_name, q.second, q.first.sparse_id);
   }
 
+  auto &swarm_container = swarm_data.Get();
+  swarm_container->SetBlockPointer(pmb);
+  for (auto const &q : resolved_packages->AllSwarms()) {
+    swarm_container->Add(q.first, q.second);
```

Comment:
As we discussed, this may need to be modified to support `Metadata::OneCopy` for `Swarm`s.

Reply:
Agreed.  I still need to address the potential consequences of moving `swarm_data` to `MeshBlockData`.  Not only with RK substaging but also partitioning.

Reply:
What happens currently if I copy a `MeshBlockData` object? I assume swarms just simply aren't copied? Agree with @brryan that in the fullness of time it would be nice to just treat them as `OneCopy` (or even make copies of them) but this is fine for now IMO.

Reply:
I now currently require that Swarms are accessed through the `base` `MeshData` register.  Is there any possible scenario where this is not adequate?